frontend/src/lib/components/DateFilter/dateFilterLogic.ts

```diff
@@ -5,6 +5,10 @@ import { isDate, dateFilterToText, dateStringToDayJs, formatDateRange, formatDat
 import { DateMappingOption } from '~/types'
 import { DateFilterLogicProps, DateFilterView } from 'lib/components/DateFilter/types'
 
+export const CUSTOM_OPTION_KEY = 'Custom'
+export const CUSTOM_OPTION_VALUE = 'No date selected'
```

Comment:
Looking at the snapshot updates, I read "No date selected" as meaning "all-time". Maybe "No date range override" to clarify that this is about the dashboard-level override?